pylds/base_discrete.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from operator import itemgetter
+from pylds.base import generate_points, lagrangian_descriptor
+
+def check_if_points_escape_box(u, box_boundaries):
```

Comment:
Isn't this function already included in base.py?

Reply:
Same name only. 

The function living in `base.py` is applicable to continuous systems since`box_boundaries` are only defined in configuration space. Plus, it's written to deal with systems of more than 1DoF. 

The function in `base_discrete.py` is a simpler and more readable version, that works fine for 2D discrete systems, which are only considered atm. 

We can merge these functions, but the cost of doing so will be that the function would become chunkier and the logic behind it would be obscured.